docs/guidelines/Colors.md

```diff
@@ -7,46 +7,37 @@ title: Colors
 
 **Page Summary**
 
-* Table of contents
-{:toc}
+* [Specifications references](#specifications-references)
+* [Implementation in Jetpack Compose](#implementation-in-jetpack-compose)
 
 ---
 
-
 ## Specifications references
 
 - [Design System Manager - Colour](https://system.design.orange.com/0c1af118d/p/623630-colour/b/041102)
 - [Material Design - The color system](https://material.io/design/color/the-color-system.html#color-usage-and-palettes)
 
 ## Implementation in Jetpack Compose
 
-### Theme colors
+As ODS library supports multi-theme, the available colors are managed by a class (`OdsColors`) which is instantiated by the theme itself. In other words, the theme provides colors for the application which uses the ODS library. ODS library components relies on the `OdsColors` defined by the used theme.
+
+The `OdsColors` class contains Material colors, functional colors and some specific components colors which allow the override of default Material behavior. For example, in Material a top app bar has a primary color background. With ODS library you can choose another color for your top app bar background and this will be defined in your theme components colors.
 
-ODS library provides MaterialTheme colors. You can use these colors by using `MaterialTheme.colors`:
+ODS library colors can be used by calling `OdsTheme.colors`:
 
 ```kotlin
-Text(
-    text = "Hello world",
-    color = MaterialTheme.colors.primary
+Icon(
+    painter = painterResource(id = R.drawable.ic_chevron_down),
+    contentDescription = null,
+    tint = OdsTheme.colors.onSurface
 )
 ```
 
-### Functional colors
-
-Functional colors have been added and can also be used as above:
-- `MaterialTheme.colors.functionalPositive`
-- `MaterialTheme.colors.functionalInfo`
-- `MaterialTheme.colors.functionalAlert`
-
-Note: These colors will be different depending on whether they are displayed in light or in dark mode.
+You can use the following **functional colors** in your application:
 
-### Other colors from Orange Design System
+- `OdsTheme.colors.functional.positive` and `OdsTheme.colors.functional.onPositive`
+- `OdsTheme.colors.functional.info`
```

Comment:
`OdsTheme.colors.functional.negative` and `OdsTheme.colors.functional.onNegative` are missing.